ansible/accumulo.yml

```diff
@@ -53,16 +53,18 @@
       when: accumulo_major_version == '2' and not use_systemd
 - hosts: accumulomaster
   tasks:
-    - name: "start accumulo 2.0 master, monitor, gc & tracer"
+    - name: "start accumulo 2.0 master, monitor, & gc"
       command: "nohup {{ accumulo_home }}/bin/accumulo-service {{ item }} start"
       register: start_result
       changed_when: "'Starting' in start_result.stdout"
       with_items:
         - master
         - monitor
         - gc
-        - tracer
       when: accumulo_major_version == '2' and not use_systemd
+    - name: "start accumulo 2.0 tracer"
+      command: "nohup {{ accumulo_home }}/bin/accumulo-service tracer start"
+      when: accumulo_version is version('2.0.0','>=') and accumulo_version is version('2.1.0','<') and not use_systemd
 - hosts: accumulomaster
   tasks:
     - name: "install and start all the accumulo services using systemd"
```

Comment:
There will be similar set of `when` conditions needed for the systemd unit for tracer. It would be good to address that as well in this PR, or maybe log an Issue to track the future change needed.

Reply:
Yes, that would be needed. If you expand this file, it's on line 73/75 below.

Reply:
I changed the systemd code later in the file but didn't test it (I did run muchos setup again and that was ok, but did not try enabling systemd).

Reply:
@keith-turner - I did test your latest changes with accumulo-2.1.0-SNAPSHOT built off accumulo main, with systemd enabled. While setup did complete without any errors, the observations from @brianloss need to be addressed sometime - especially the accumulo-cluster-systemd script errors out.

BTW, unrelated to this change, the accumulo-cluster-systemd also errors out due to lack of change to `manager` from the older term. That is an unrelated issue and can be cleaned up separately.

Reply:
> BTW, unrelated to this change, the accumulo-cluster-systemd also errors out due to lack of change to manager from the older term. That is an unrelated issue and can be cleaned up separately.

I made one small change related to this in daeab54 because I was touching that code.  Looking at the file its pervasive , so would be nicer to do that in its own PR.  I wonder if we can avoid a bunch of if stmts in the template and just have template variable that resolves to manager or master.